php-web-form/README.md

```diff
@@ -0,0 +1,20 @@
+# SalesLoft API - PHP Web Form Handler
+
+This guide will look at how a PHP web form might be handled to add a person to the SalesLoft
+platform, and then add the newly created person to a specific inbound cadence. PHP is chosen
+as it is a very common language for marketing sites to be implemented in.
+
+## Setup
+
+1. Generate an API key at https://accounts.salesloft.com/api_keys/new, or use an existing key. Treat this
+   key like you would a password.
+2. Create a cadence at https://app.salesloft.com/app/cadences_v2/new with a step on it, or use an existing cadence.
+   Use the cadence id in the URL in order to know which cadence to add to.
+3. `CADENCE_ID=FROM_APP API_KEY=FROM_ACCOUNTS_SALESLOFT_COM php -d variables_order=EGPCS -S localhost:5001`
```

Comment:
TIL macs have PHP installed by default

Reply:
this is awesome 
![image](https://user-images.githubusercontent.com/16196910/33272909-acf42cb2-d359-11e7-8f44-990b684bd2a1.png)

Reply:
⭐️  <- for QA

Reply:
Does this require anything that doesn't come on a default mac? Like something that would've gotten snuck in for me on boxen, but would need to be downloaded by someone trying this out

Reply:
I think it's good to go